hack/update-generated-swagger-docs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
If someone updates `types.go` then for sure we must update swagger specs. So I agree we might want to append swagger spec update after swagger doc update.

On the other hand we need swagger spec update as a separate script as there are also other reasons that the spec might change. (e.g. alter tags, change go-restful, etc..)

So I will add a call to update specs inside update docs.